concent_api/core/admin.py

```diff
@@ -48,17 +60,27 @@ class SubtaskAdmin(ModelAdminReadOnlyMixin, admin.ModelAdmin):
         'task_id',
     ]
 
-    def get_provider_public_key(self, obj):  # pylint: disable=no-self-use
+    @classmethod
+    def get_provider_public_key(cls, obj):
```

Comment:
This change is not related to filters. It's fixing a pylint warning. It should be in a separate commit.

Reply:
The commit description is still misleading. I see now that you're not fixing warnings. You're just removing the unnecessary `disable=no-self-use`. So why not describe it as
```
Replace 'pylint: disable=no-self-use' with the use of @classmethod in SubtaskAdmin
```
?